crawlers/mooncrawl/mooncrawl/blockchain.py

```diff
@@ -70,6 +71,8 @@ def connect(
             web3_uri = MOONSTREAM_MUMBAI_WEB3_PROVIDER_URI
         elif blockchain_type == AvailableBlockchainType.XDAI:
             web3_uri = MOONSTREAM_XDAI_WEB3_PROVIDER_URI
+        elif blockchain_type == AvailableBlockchainType.WYRM:
+            web3_uri = MOONSTREAM_WYRM_WEB3_PROVIDER_URI
```

Comment:
@kompotkot need to push update in robots to change `MOONSTREAM_CALDERA_WEB3_PROVIDER_URI` -> `MOONSTREAM_WYRM_WEB3_PROVIDER_URI` and update env variable